core/tracepoint/inspect_spec.rb

```diff
@@ -0,0 +1,10 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+
+ruby_version_is '2.0' do
+  describe 'TracePoint#inspect' do
+    it 'return a string containing a human-readable TracePoint status' do
+      TracePoint.new(:call) {}.inspect.should ==
```

Comment:
This spec is a bit too precise. In particular, we want to give other implementations the possibility to add information. See specs for `Dir#inspect`